Makefile

```diff
@@ -10,7 +10,7 @@ BIN := .tmp/bin
 CACHE := .tmp/cache
 LICENSE_HEADER_YEAR_RANGE := 2022-2023
 LICENSE_HEADER_VERSION := v1.28.1
-CONFORMANCE_VERSION := v1.0.0-rc1
+CONFORMANCE_VERSION := v1.0.0-rc2
```

Comment:
The main change this brings in is a new method on the conformance service: `IdempotentUnary`, which is just like `Unary`, but it is declared to be side-effect-free so it can be used with Connect GET.